drivers/hmis/lib/form_data/allegheny/fragments/client_mci.json

```diff
@@ -1,6 +1,5 @@
 {
   "type": "GROUP",
-  "required": false,
```

Comment:
I think 'required' doesn't do anything on a group. These are probably left in accidentally(?)